src/pages-and-resources/open_responses/Settings.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import AppSettingsModal from '../app-settings-modal/AppSettingsModal';
+import messages from './messages';
+
+const OpenResponsesSettings = ({ intl, onClose }) => (
+  <AppSettingsModal
```

Comment:
What we'd like here is to make a change to AppSettingsModal where we add a parameter to be able to hide the "enable/disable app" switch. 

This "app" will always be enabled. Then, add another boolean field to this modal to enable / disable the specific setting. See `wiki/Settings.jsx`

Reply:
After discussion with Daniel and Gab, I decide to keep the code as is. I will however rename it to flexible peer grading to avoid confusion.